src/MoBi.R/Services/MoBiRIntegrationService.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using MoBi.Core.Domain.Model;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Core.Domain.Data;
+using OSPSuite.Core.Domain.ParameterIdentifications;
+using ISerializationTask = MoBi.Presentation.Tasks.ISerializationTask;
+
+namespace MoBi.R.Services
+{
+   public interface IMoBiRIntegrationService
+   {
+      IndividualBuildingBlock CreateIndividual(string name);
+      void SetParameterValue(ParameterValuesBuildingBlock pvBB, string parameterName, double newValue);
+   }
+
+   public class MoBiRIntegrationService : IMoBiRIntegrationService
```

Comment:
We usually use Services as the namespace, but the objects and interfaces end with `Task`.

What's the plan for this one, because it looks like it's just going to be a set of unrelated one-off methods, but I'd still favor separate tasks that implement few public methods instead of one that does a lot of unrelated stuff.